bot/admin/web/src/app/bot/flow/flow.component.html

```diff
@@ -18,29 +18,32 @@
   <nb-accordion-item #accordion>
     <nb-accordion-item-header>
       Bot Flow
-      <span *ngIf="statsMode" class="status status-header statsMode-badge statsMode-true" matTooltip="Dynamic mode (user flow)">Dynamic</span>
-      <span *ngIf="!statsMode" class="status status-header statsMode-badge statsMode-false" matTooltip="Static mode (available stories)">Static</span>
-      <mat-icon *ngIf="accordion.collapsed" matTooltip="Show options">expand_more</mat-icon>
-      <mat-icon *ngIf="!accordion.collapsed" matTooltip="Hide options">expand_less</mat-icon>
+      <span *ngIf="statsMode" class="status status-header statsMode-badge statsMode-true" title="Dynamic mode (user flow)">Dynamic</span>
+      <span *ngIf="!statsMode" class="status status-header statsMode-badge statsMode-false" title="Static mode (available stories)">Static</span>
+      <mat-icon *ngIf="accordion.collapsed" title="Show options">expand_more</mat-icon>
+      <mat-icon *ngIf="!accordion.collapsed" title="Hide options">expand_less</mat-icon>
```

Comment:
nb-icon

Reply:
and also nbTooltip. I merge the PR, we need to describe how we will switch completely to nebula for 20.9.9